### PR TITLE
DSEAT-35 Implemented functionality to disable split bill add button a…

### PR DESCRIPTION
**Pull request by IremA99**
https://re-motion.atlassian.net/jira/software/projects/DSEAT/boards/17?selectedIssue=DSEAT-35

### CODE REVIEW

**Review by FelixSchoberRubi — APPROVED** [review]

**Review by FelixSchoberRubi — CHANGES_REQUESTED** [review]
Wait for the branch for the remove button to be added, and then rebase this branch on main and adapt it so that removing a person enables the add button again

**Review by FelixSchoberRubi — APPROVED** [review]

File: RubeesEat/wwwroot/js/SplitBill.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 let counter = 0;
 
 function AddPerson() {
+    const dropdown = document.getElementById("billPeople");
```
Comment (KuJo7) [inline RubeesEat/wwwroot/js/SplitBill.js:4]:
```suggestion
    const dropdown = document.getElementById("billPeople");
    const AddPerson = document.getElementById("AddPerson");
```

File: RubeesEat/wwwroot/js/SplitBill.js [hunk 2]
```diff
@@ -35,12 +36,20 @@ function AddPerson() {
         addToOptions.dataset.id = selectedId;
         addToOptions.textContent = selectedPerson;
         selectElement.appendChild(addToOptions);
+
+        document.querySelector("#addPersonButton").disabled = false;
+        dropdown.hidden = false;
```
Comment (KuJo7) [inline RubeesEat/wwwroot/js/SplitBill.js:41]:
```suggestion
        AddPerson.hidden = false;
```

File: RubeesEat/wwwroot/js/SplitBill.js [hunk 3]
```diff
@@ -35,12 +36,20 @@ function AddPerson() {
...
     personDiv.appendChild(button);
     document.getElementById("addedPeople").appendChild(personDiv);
-    selectElement.remove(selectElement.selectedIndex);
     
     counter++;
+
+    dropdown.remove(dropdown.selectedIndex)
+    if (selectElement.options.length === 0) {
+        document.querySelector("#addPersonButton").disabled = true;
+        dropdown.hidden = true;
```
Comment (KuJo7) [inline RubeesEat/wwwroot/js/SplitBill.js:51]:
```suggestion
        AddPerson.hidden = false;
```

**Review by KuJo7 — CHANGES_REQUESTED** [review]
Put the part for addPerson in a separate div with the id `AddPerson`:
```
    <div id="AddPerson">
        <label for="billPeople">Add Person:</label>
        <select name="billPeople" id="billPeople">
            @foreach (Person person in PersonRepository.GetAll())
            {
                <option value="@person.FirstName @person.LastName" data-id="@person.Id">@person.FirstName @person.LastName </option>
            }
        </select>
        <input id="addPersonButton" type="button" onclick="AddPerson()" value="Add Person"><br/>
    </div>
```

**Review by KuJo7 — APPROVED** [review]

**Review by a-ctor — APPROVED** [review]

**Pull request merged**